File: aoj/volume0/n0024/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        while (sc.hasNext()) {

            double velocity = sc.nextDouble();

            double time = velocity / 9.8;

            double fallDistance = 4.9 * time * time;

            int floorCount = 1;

            while (true) {

                double height = 5 * floorCount - 5;

                if (fallDistance <= height) {
                    System.out.println(floorCount);
                    break;
                }

                floorCount++;

            }

        }

    }

}

File: aoj/volume0/n0024/Main.cpp
#include <iostream>

using namespace std;

int main() {

    double velocity;

    while (cin >> velocity) {

        double time = velocity / 9.8;
        double fallDistance = 4.9 * time * time;

        int floorCount = 1;

        while (1) {

            double height = 5 * floorCount - 5;

            if (fallDistance <= height) {
                break;
            }

            floorCount++;

        }

        cout << floorCount << endl;

    }

    return 0;

}
